docs/core/diagnostics/dotnet-trace.md

```diff
@@ -25,17 +25,22 @@ dotnet-trace [-h, --help] [--version] <command>
 
 ## Description
 
-The `dotnet-trace` tool is a cross-platform CLI global tool that enables the collection of .NET Core traces of a running process without any native profiler involved. It's built around the cross-platform `EventPipe` technology of the .NET Core runtime. `dotnet-trace` delivers the same experience on Windows, Linux, or macOS.
+The `dotnet-trace` tool:
+
+* Is a cross-platform CLI global tool.
```

Comment:
I think we're going to start moving away from global and local tool terminology. @KathleenDollard can you confirm this is the right terminology now?

```suggestion
* Is a cross-platform .NET Core tool.
```

Reply:
@mairaw Fine with me - go ahead and commit.